.mailmap

```diff
@@ -0,0 +1,18 @@
+=^._.^= <max@maxogden.com> <maxogden@users.noreply.github.com>
+Alexey Kuzmin <alex.s.kuzmin@gmail.com> <alexeykuzmin@users.noreply.github.com>
+Alexey Kuzmin <alex.s.kuzmin@gmail.com> <alkuzmin@microsoft.com>
+Aleš Pergl <alpergl@microsoft.com> <alespergl@users.noreply.github.com>
+Amos Wenger <fasterthanlime@gmail.com> <fasterthanlime@users.noreply.github.com>
+Destan Sarpkaya <destan@dorukdestan.com> <destan@users.noreply.github.com>
+Gabriel Handford <gabrielh@gmail.com>
+Ivan Mir <ivmirx@fastmail.fm> <ivmirx@users.noreply.github.com>
+Jacob Groundwater <groundwater@gmail.com> <groundwater@github.com>
+João Moreno <mail@joaomoreno.com> <joaomoreno@users.noreply.github.com>
+Machisté N. Quintana <mnquintana@users.noreply.github.com>
+Samuel Attard <samuel.r.attard@gmail.com> <samuel.attard@orionhealth.com>
+Samuel Attard <samuel.r.attard@gmail.com> <sattard@atlassian.com>
+Sergey Rubanov <chi187@gmail.com>
+Shelley Vohr <shelley.vohr@gmail.com> <codebytere@github.com>
```

Comment:
@MarshallOfSound, are you ok having your gmail address as a canonical one?

Reply:
lord how do i have this many emails

Reply:
csvohr1995@gmail.com, yeah

Reply:
@codebytere Done.